cmd/helm-sync/main.go

```diff
@@ -119,7 +127,14 @@ func main() {
 			UserName:        *flUsername,
 			Password:        *flPassword,
 		}
-		if err := hydrator.HelmTemplate(ctx); err != nil {
+
+		var refreshVersion bool
```

Comment:
minor nit: assign this as `refreshVersion := false`. It's functionally the same but makes the intent a bit clearer IMO

Reply:
done